dark/index.html

```diff
@@ -312,7 +313,7 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
       <!-- Contact Section -->
 
       <section id="contact-section">
-        <h1>CONTACT ME</h1>
+      <h1>CONTACT ME</h1>

```

Comment:
Remove this title bcs already there is a title @muskaan04

Reply:
Changes done @YoshithaRathnayake